### PR TITLE
Parity Support

### PR DESCRIPTION
**Pull request by tagomaru**
This PR is for parity support.
It needs to change the repos `priv-testnet`, `orchestration`, `relay`, and `polyswarmd` as well.

### CODE REVIEW

File: src/contractor/network.py [hunk 1]
```diff
@@ -164,11 +164,15 @@ def __preflight_checks(self):
             raise Exception('Connected to network with incorrect network id')
 
         cur_block = start_block = self.w3.eth.blockNumber
-        while cur_block < BLOCKS_TO_WAIT or cur_block - start_block < BLOCKS_TO_WAIT:
-            logger.info('Waiting for blocks to advance')
-            time.sleep(1)
-
+        
+        if self.chain == Chain.SIDECHAIN:
             cur_block = self.w3.eth.blockNumber
```
Comment (myhndl) [inline src/contractor/network.py]: Why assign `cur_block` to `self.w3.eth.blockNumber` twice in this path?

Comment (tagomaru) [reply src/contractor/network.py]: Ah sorry, it was not necessary.
It was removed on the next commit.

**Comment by tagomaru** [comment]
> wait only when chain is side and sidechain is parity.
Sorry this is my mistake. 
I meant "wait **EXCEPT** when chain is side and sidechain is parity".

**Comment by tagomaru** [comment]
@Rizato 
Yes, thanks to PoA, this is needed now.
Thx!

**Comment by myhndl** [comment]
> @Rizato
> Yes, thanks to PoA, this is needed now.
> Thx!

Do you mean this is *not* needed now?

If this PR is not needed, please close it.

**Comment by tagomaru** [comment]
I am going to close this thanks to PoA.

**Pull request closed without merging**